api/controllers/PublicController.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
           throw new Error('Email not recognised.')
         } else {
           randomString = ForgotPass.randomString()
-          return Members.update({ id: member.id }, { new_password: randomString })
+          return Members.update({ id: member.id }, { new_password: randomString, id: member.id })
```

Comment:
I just took this out and it broke tests again. It is not here that it is used to find a member, it is in `handle_membership_change` in the Members model. That function only gets the fields that are being changed and would only get `new_password` in this case. Therefore it would not be able to find the member to check for possible `membership_type` change.